SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Ingestion/Models/HandledErrorLog.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.AppCenter.Crashes.Ingestion.Models
+{
+    using Microsoft.AppCenter.Ingestion.Models;
+    using Newtonsoft.Json;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Handled Error log for managed platforms (such as Xamarin, Unity,
+    /// Android Dalvik/ART)
```

Comment:
```suggestion
    /// Android Dalvik/ART).
```

Reply:
We don't want to change generated models. This should be changed in YAML.